{
  try {
    Assert.assertEquals('a',((Number)this.extractor.getValue(this.bean)).intValue());
  }
 catch (  final Exception e) {
    fail("Should not throw an exception");
  }
}
